pom.xml

```diff
@@ -216,7 +230,7 @@
         <typesafe.config.version>1.3.0</typesafe.config.version>
         <lombok.version>1.18.0</lombok.version>
         <cleartk.version>2.0.0</cleartk.version>
-        <lucene-solr.version>7.4.0</lucene-solr.version>
+        <lucene-solr.version>7.5.0-SNAPSHOT</lucene-solr.version> <!-- TODO: once it's released change this to 7.5.0 for https://issues.apache.org/jira/browse/SOLR-12402 change -->
```

Comment:
pull request cross-reference: i opened separate #6485 for the 7.4.0 to 7.5.0 upgrade since it involves the one-line pom.xml change here plus a couple of java doc updates elsewhere.